src/UsbUtil.cpp

```diff
@@ -86,33 +86,71 @@ llvm::StringRef GetUsbNameFromId(int vendor, int product,
     case 0x046d:
       os << "Logitech, Inc. ";
       switch (product) {
-        case 0x0802: os << "Webcam C200"; break;
-        case 0x0804: os << "Webcam C250"; break;
-        case 0x0805: os << "Webcam C300"; break;
-        case 0x0807: os << "Webcam B500"; break;
-        case 0x0808: os << "Webcam C600"; break;
-        case 0x0809: os << "Webcam Pro 9000"; break;
-        case 0x080a: os << "Portable Webcam C905"; break;
-        case 0x080f: os << "Webcam C120"; break;
-        case 0x0819: os << "Webcam C210"; break;
-        case 0x081b: os << "Webcam C310"; break;
-        case 0x081d: os << "HD Webcam C510"; break;
-        case 0x0821: os << "HD Webcam C910"; break;
-        case 0x0825: os << "Webcam C270"; break;
-        case 0x0826: os << "HD Webcam C525"; break;
-        case 0x0828: os << "HD Webcam B990"; break;
-        case 0x082b: os << "Webcam C170"; break;
-        case 0x082d: os << "HD Pro Webcam C920"; break;
-        case 0x0836: os << "B525 HD Webcam"; break;
-        case 0x0843: os << "Webcam C930e"; break;
+        case 0x0802:
+          os << "Webcam C200";
+          break;
+        case 0x0804:
+          os << "Webcam C250";
+          break;
+        case 0x0805:
+          os << "Webcam C300";
+          break;
+        case 0x0807:
+          os << "Webcam B500";
+          break;
+        case 0x0808:
+          os << "Webcam C600";
+          break;
+        case 0x0809:
+          os << "Webcam Pro 9000";
+          break;
+        case 0x080a:
+          os << "Portable Webcam C905";
+          break;
+        case 0x080f:
+          os << "Webcam C120";
+          break;
+        case 0x0819:
+          os << "Webcam C210";
+          break;
+        case 0x081b:
+          os << "Webcam C310";
+          break;
+        case 0x081d:
+          os << "HD Webcam C510";
+          break;
+        case 0x0821:
+          os << "HD Webcam C910";
+          break;
+        case 0x0825:
+          os << "Webcam C270";
+          break;
+        case 0x0826:
+          os << "HD Webcam C525";
+          break;
+        case 0x0828:
+          os << "HD Webcam B990";
+          break;
+        case 0x082b:
+          os << "Webcam C170";
+          break;
+        case 0x082d:
+          os << "HD Pro Webcam C920";
+          break;
+        case 0x0836:
+          os << "B525 HD Webcam";
+          break;
+        case 0x0843:
+          os << "Webcam C930e";
+          break;
       }
       break;
   }
 
   return os.str();
 }
 
-int CheckedIoctl(int fd, unsigned long req, void* data, const char* name,
+int CheckedIoctl(int fd, uint32_t req, void* data, const char* name,
```

Comment:
Oh, the POSIX and Linux man pages for ioctl differ. POSIX says int while Linux says unsigned long.